modules/dxp/apps/search-experiences/search-experiences-test/src/testIntegration/java/com/liferay/search/experiences/internal/blueprint/test/SXPBlueprintSearchResultTest.java

```diff
@@ -223,10 +223,22 @@ public void testBoostContentsForTheCurrentLanguage() throws Exception {
 			"coca cola es_ES"
 		).setContent(
 			"cola"
+		).setTitleMap(
+			HashMapBuilder.put(
+				LocaleUtil.SPAIN, "coca cola es_ES"
+			).put(
+				LocaleUtil.US, "coca cola es_ES"
```

Comment:
Hello almir, the ES means that the title is in Spanish, so that we can identify it in the test, the title map is there so that we can translate something from English into Spanish.

Reply:
https://liferay.slack.com/archives/CLBEAPQ5D/p1698715397403009?thread_ts=1698697303.201999&cid=CLBEAPQ5D @AlmirFe Brian Lee tested in a enviroment to check if the flaky is still happening and looks good, but its difficult to have 100% sure, after merged we will check how this change will behaves with other tests.

Reply:
Hi @felipelorenz, I'm not sure if this is the best solution to this problem, but I believe we can better understand this problem if we test this solution. So let's forward this and see how it performs, just like Brian Lee said in the message.